docker-compose.yml

```diff
@@ -58,60 +58,58 @@ services:
-      # in the expected location and the environment variable KAFKA_CFG_SSL_KEYSTORE_LOCATION
-      # points to the real keystore when the kafka server is started.
-      - ./kafka/kafka.keystore.jks:/bitnami/kafka/config/certs/kafka.keystore.jks
-      - ./kafka/kafka.properties:/bitnami/kafka/config/kafka.properties
+      - keystore:/etc/kafka/secrets/keystore
+      - ./kafka/update_run.sh:/tmp/update_run.sh
+      - ./kafka/kafka.properties:/tmp/kafka.properties
     networks:
       - callisto
     environment:
-      KAFKA_ENABLE_KRAFT: yes
-      KAFKA_CFG_PROCESS_ROLES: broker,controller
-      KAFKA_CFG_CONTROLLER_LISTENER_NAMES: CONTROLLER
-      KAFKA_CFG_LISTENERS: CONTROLLER://kafka:9092,SSL://kafka:9093
-      KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP: CONTROLLER:SSL,SSL:SSL
-      KAFKA_CFG_ADVERTISED_LISTENERS: SSL://kafka:9093
+      KAFKA_ADVERTISED_LISTENERS: SSL://kafka:9093
+      KAFKA_ALLOW_EVERYONE_IF_NO_ACL_FOUND: false
+      KAFKA_AUTHORIZER_CLASS_NAME: org.apache.kafka.metadata.authorizer.StandardAuthorizer
+      KAFKA_AUTO_CREATE_TOPICS_ENABLE: false
       KAFKA_BROKER_ID: 1
-      KAFKA_CFG_CONTROLLER_QUORUM_VOTERS: 1@kafka:9092
-
-      KAFKA_CFG_INTER_BROKER_LISTENER_NAME: SSL
-      KAFKA_CFG_SSL_TRUSTSTORE_LOCATION: /keystore/kafka/kafka.truststore.jks
-      KAFKA_CFG_SSL_TRUSTSTORE_PASSWORD: changeit
-      KAFKA_TLS_TRUSTSTORE_FILE: /keystore/kafka/kafka.truststore.jks
-      KAFKA_CERTIFICATE_PASSWORD: changeit
-      KAFKA_CFG_SSL_KEYSTORE_LOCATION: /keystore/kafka/kafka.keystore.jks
-      KAFKA_CFG_SSL_KEYSTORE_PASSWORD: changeit
-
-      KAFKA_CFG_AUTHORIZER_CLASS_NAME: org.apache.kafka.metadata.authorizer.StandardAuthorizer
-      KAFKA_CFG_ALLOW_EVERYONE_IF_NO_ACL_FOUND: false
-      KAFKA_CFG_SSL_CLIENT_AUTH: required
-      KAFKA_CFG_SSL_PRINCIPAL_MAPPING_RULES: RULE:^C=GB,O=UK Home Office,CN=Callisto (.+)$$/$$1/,DEFAULT
-      KAFKA_CFG_SUPER_USERS: "User:kafka"
+      KAFKA_CONTROLLER_LISTENER_NAMES: CONTROLLER
+      KAFKA_CONTROLLER_QUORUM_VOTERS: 1@kafka:9092
+      KAFKA_INTER_BROKER_LISTENER_NAME: SSL
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: CONTROLLER:SSL,SSL:SSL
+      KAFKA_LISTENERS: CONTROLLER://kafka:9092,SSL://kafka:9093
+      KAFKA_LOG_DIRS: '/tmp/kraft-combined-logs'
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+      KAFKA_PROCESS_ROLES: broker,controller
+      KAFKA_SSL_CLIENT_AUTH: required
+      KAFKA_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM: " "
+      KAFKA_SSL_KEY_CREDENTIALS: keystore/kafka/kafka_creds
+      KAFKA_SSL_KEYSTORE_CREDENTIALS: keystore/kafka/kafka_creds
+      KAFKA_SSL_KEYSTORE_FILENAME: keystore/kafka/kafka.keystore.jks
+      KAFKA_SSL_PRINCIPAL_MAPPING_RULES: RULE:^C=GB,O=UK Home Office,CN=Callisto (.+)$$/$$1/,DEFAULT
+      KAFKA_SSL_TRUSTSTORE_CREDENTIALS: keystore/kafka/kafka_creds
+      KAFKA_SSL_TRUSTSTORE_FILENAME: keystore/kafka/kafka.truststore.jks
```

Comment:
These are relative paths. Is this intended?

Reply:
> Yes, the setup in this image appends these paths to a predefined location which is where the keystore volume is mounted for this service `/etc/kafka/secrets` https://github.com/UKHomeOffice/callisto-localdev/pull/15/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R65